libs/eo/core/shell/src/lib/eo-core-shell.module.ts

```diff
@@ -65,6 +66,16 @@ const routes: Routes = [
             (esModule) => esModule.EoDashboardShellModule
           ),
       },
+      {
+        path: eoOriginOfEnergyRoutePath,
+        data: {
+          title: 'Origin of Energy on the hourly declaration',
```

Comment:
question: Is this what the page title in the top app bar is supposed to say?

Reply:
This is what has been proposed by UX yeah 😄

Reply:
Ask Kenneth whether he wants to commit to this title and maybe the same for other page title suggestions?

Reply:
He says this title should stick 😄

Reply:
Just updated the title because "based" was missing 😄